cypress/e2e/app_tests/Register_page.Check_fields_validation.cy.js

```diff
@@ -0,0 +1,42 @@
+//Валідація 2-3 полів, потрібно ввести текст який не відповідає патерну для цього поля, 
+//та перевірити ерор меседж. 
+//Для того щоб побачити ерор меседж, у тесті можна клікнути на боді(cy.get("body").click()), 
+//або на будь який інший елемент Для більшої наглядності додаю скрін валідація ерор меседжу
+
+/// <reference types="cypress" />
+
+describe('Fields validation. Should have correct error message', () => {
+    beforeEach(() => cy.visit('/auth/register'));
+
+    it('Empty fields', () => {
+        cy.get('#input-email').click();
+        cy.get('#input-password').click();
+        cy.get('#input-re-password').click();
+        cy.get('#input-name').click();
+        cy.get('.appearance-filled').should('be.disabled');
+        cy.get(':nth-child(2) > .caption').should('have.text', " Email is required! ");
+        cy.get(':nth-child(3) > .caption').should('have.text', " Password is required! ");
+        cy.get(':nth-child(4) > .caption').should('have.text', " Password confirmation is required! ");
```

Comment:
small improvements here
```suggestion
        cy.get('.caption').should(($el) => {
            expect($el[0]).to.have.text(" Email is required! ");
            expect($el[1]).to.have.text(" Password is required! ");
            expect($el[2]).to.have.text(" Password confirmation is required! ");
        });
```